.github/workflows/changelog/changelog

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A simple changelog generator
+
+NOTE: This is a repo-specific script, so you may not use it in other places.
+
+e.g.
+    cd spark-rapids/
+    .github/workflows/changelog/changelog --token=<GITHUB_PERSONAL_ACCESS_TOKEN> --base_refs=branch-0.1,branch-0.2,branch-0.3
```

Comment:
@tgravescs yes, plz take a look at the [CHANGELOG.md](https://gist.github.com/pxLi/2e0b148c74baf951234a4ad88d8d2494), this is the rendered markdown preview. Since we separate prs&issues by the release info (`github projects`), so I try to map the pr&issue back to projects info. And plz let me know if anything else you want

Reply:
my comment was saying we should update the description of this script to tell user what base_ref really does.

Reply:
@tgravescs sure thing. I will add a detailed description in upcoming commits